homeassistant/components/generic_thermostat/__init__.py

```diff
@@ -3,13 +3,23 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device import (
+    async_remove_stale_devices_links_keep_entity_device,
+)
 
+CONF_HEATER = "heater"
```

Comment:
Moving `CONF_HATER` to `__init__.py` is necessary to avoid circular import between `__init__.py` and `climate.py`.

If you wish, I can create another PR to move all constants shared between `__init__.py`, `climate.py` and `confg flow.py` to a `const.py` file.

Reply:
Yes i think that would make sense.